external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
-# Inspect Command
+# inspect command
+The inspect command is used to inspect the metadata of an existing Qube.
 
-todo: Complete me!
\ No newline at end of file
+**Syntax**  
+`csvcubed inspect [OPTIONS] TIDY_CSV_PATH`
+
+**Arguments**
+
+| Argument      | Description                                                                                        |
+| ------------- | -------------------------------------------------------------------------------------------------- |
+| TIDY_CSV_PATH | The file path to the Qube data file, formatted as [tidy data](../glossary/index.md#tidy-data) csv. |
+
+**Options**
+
+| Option      | Description                                                                                                      |
+| ----------- | ---------------------------------------------------------------------------------------------------------------- |
+| --help / -h | Show the command help text.                                                                                      |
+| --log-level | Set the desired logging level to one of 'crit', 'err', 'warn', 'info' and 'debug'.  <br/> The default is 'warn'. |
+
+**Log output**  
```

Comment:
Can you create a separate file in the `command-line` folder which deals with explaining the log location? Ideally then both pages can point to that instead of repeating eachother.